### PR TITLE
add information about how we work

### PR DESCRIPTION
**Pull request by afeld**
I was going to say some of these things to @adborden directly, but figured I might as well write them down.

### CODE REVIEW

File: how_we_work/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- We don't have team-wide core hours.
+- Aside from meetings, your schedule is yours.
+- You are expected to keep your calendar up-to-date.
+- We expect response to messages within a few working hours.
+- You are expected to work forty hours per week, no more, no less.
+  - If you have too much or too little to do, tell the Director.
+
+See [meetings](policy/meetings.md) for more details.
+
+## Team maxims
```
Comment (afeld) [inline how_we_work/README.md:21]: Are these useful to include? Was trying to think of things that distinguish us from other teams.

File: how_we_work/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+- Open is better than closed.
+  - For information that isn't [sensitive](https://handbook.tts.gsa.gov/sensitive-information/), make it as public as possible/practical.
+- [There's no such thing as user error.](https://medium.com/swlh/theres-no-such-thing-as-user-error-76b564454234)
+- If you see something, fix something.
+- We have way more work to do than people to do it, so we are constantly on the lookout for ways to multiply our impact and things to take off our plates.
```
Comment (adborden) [inline how_we_work/README.md:27]: I think Dave Z would disagree with this and instead we should have the people to cover the things we need to do.

That said, it's correct at the moment and is a useful framework to keep in mind during prioritization. Maybe I would rephrase 🤷 

```suggestion
- We have way more requests than we have people to respond, so we are constantly on the lookout for ways to multiply our impact and things to take off our plates.
```

**Review by adborden — APPROVED** [review]

File: how_we_work/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+See [meetings](policy/meetings.md) for more details.
+
+## Team maxims
+
+- Open is better than closed.
+  - For information that isn't [sensitive](https://handbook.tts.gsa.gov/sensitive-information/), make it as public as possible/practical.
+- [There's no such thing as user error.](https://medium.com/swlh/theres-no-such-thing-as-user-error-76b564454234)
+- If you see something, fix something.
+- We have more asked of our team than we are able to fulfill, so we are constantly on the lookout for ways to multiply our impact and things to take off our plates.
```
Comment (afeld) [inline how_we_work/README.md:27]: @adborden Better?

Comment (adborden) [reply how_we_work/README.md:27]: lgtm, thanks!
